verifier/src/lib.rs

```diff
@@ -3,13 +3,13 @@
 use air::{ProcessorAir, PublicInputs};
 use core::fmt;
 pub use vm_core::{Kernel, ProgramInfo, StackInputs};
-pub use winterfell::VerifierError;
+pub use winter_verifier::VerifierError;
 
 // EXPORTS
 // ================================================================================================
 
-pub use vm_core::{chiplets::hasher::Digest, StackOutputs, Word};
-pub use winterfell::StarkProof;
+pub use vm_core::{chiplets::hasher::Digest, crypto::hash::Blake3_192, StackOutputs, Word};
```

Comment:
Why do we need to export the hash function publicly from here?

Reply:
We don't strictly need it. That is behind the idea that we re-export everything we expose from prover and verifier. The hash primitive isn't really needed as the user will not have to declare anything with it, so it can be safely removed. Fixed!